### DISCUSSION TITLE
Charuco board print size

### DISCUSSION THREAD

**Discussion by steenharsted** [freemocap/freemocap#158] [Q&A]
I have created the charuco_board_image.png file, and I am ready to print it on a board, but what size should the physical board be?

My recording will eventually be quite large, so I have changed

`charuco_board_image = board.draw((2000,2000)) #`2000` is the resolution of the resulting image. Increase this number if printing a large board (bigger is better! Esp for large spaces!`

to 

`charuco_board_image = board.draw((5000,5000)) #`2000` is the resolution of the resulting image. Increase this number if printing a large board (bigger is better! Esp for large spaces!`

Will/should this change affect the physical size of my board?

**Comment by philipqueen** [comment]
The physical board can be any size, as long as it shows up clearly in the recordings. Bigger can be easier to use, but plenty of people have had success with boards printed on 8 1/2" x 11" paper. For smaller boards, you may have to hold them closer to the cameras for calibration, while bigger boards will allow you to stand further away in the frame(s). Whatever size you do print it, make sure it's mounted to something rigid (cardboard, gator board, taped to a hardcover book?). It will seriously affect calibration if your board isn't flat. 

As for the board image parameter, this will effect the resolution of print (bigger prints want higher resolution), but the physical size of the board will be determined by where/how you print it. A lot of people have had success at printing companies by asking for prints on gatorboard - a printing shop should be able to help print a certain size. 

Once you have your board, measure the width of one black square in mm and enter that in the RunMe as charucoSquareSize = **your square size in mm** and that will ensure the data from your recordings have correct dimensions. You can find more info on the RunMe here: https://github.com/freemocap/freemocap/blob/main/OLD_README.md
